Added docs for athena & snowflake (#1777)
flyteorg/flyte: rsts/deployment/plugin_setup/aws/athena.rst

.. _deployment-plugin-setup-aws-athena:

Athena Plugin Setup
-------------------

> This guide gives an overview of how to set up the Athena in your AWS Flyte deployment. Athena plugin need flyte deployment in aws cloud, It will not work in sandbox/GCP/Azure flyte deployment. Please provide correct service account to flyte propeller for athena access.

Reviewer: ```suggestion
This guide gives an overview of how to set up Athena in your Flyte deployment. Athena plugin needs Flyte deployment in AWS cloud; sandbox/GCP/Azure wouldn't work. Please provide the correct service account to FlytePropeller for Athena access.
```
